README.md

```diff
@@ -1,4 +1,4 @@
-**MetaDTA v2.0.5**
+**MetaDTA v2.1.0**
```

Comment:
Update link to crsu.shinyapps.io/MetaDTA (I updated the sidebar in the repo)

Reply:
Thank you, updated.

Reply:
I have also removed the link to metadta v1, as this no longer exists (it just redirects to this version),